R/si_gravity.R

```diff
@@ -0,0 +1,27 @@
+si_gravity <- function(O_i, D_j, C_ij, flows, data,
```

Comment:
Can you try adding some examples? You can use the "Insert Roxygen Skeleton" button, in the Code tab if I remember, to do this I think.